chaco/tests/test_scatter_1d.py

```diff
@@ -0,0 +1,139 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+"""
+Unit tests for scatter_1d function
+"""
+
+from numpy import sort
+from numpy.random import random, randint, uniform
+import unittest
+from chaco.api import ArrayPlotData, Plot
+
+
+class Scatter1DTestCase(unittest.TestCase):
+    # test the default value 4.0
+    def test_default(self):
+        # Create some data
+        numpts = 50
+        x = sort(random(numpts))
+        y = random(numpts)
+
+        # Create a plot data object and give it this data
+        pd = ArrayPlotData()
+        pd.set_data("index", x)
+        pd.set_data("value", y)
+
+        # Create the plot
+        plot = Plot(pd, use_backbuffer=True, auto_grid=False)
+
+        plot.plot_1d(
+            "value",
+            type="scatter_1d",
+            orientation="v",
+            marker="plus",
+            alignment="left"
+        )
```

Comment:
that is applicable to all the tests.

Reply:
Yes, these tests are limited to check for traits mismatch exceptions. Is there other stuffs we want to test?

Reply:
If this is a test of the default marker size, you should somewhere have a line like `self.assertEqual(..., 4.0)`

With the code as you have it now, it would fail!